docs/devGuide/workflow.md

```diff
@@ -126,6 +126,22 @@ The sections below has more information about various stages of submitting a PR.
 
   </modal>
 
+#### Editing backend features
```

Comment:
let's update the release guide as well with the relevant additional steps

Reply:
As far as I remember, we don't need to do any additional things for release.

I have set core's npm lifecycle script `prepare` which will run as part of [`npm publish`](https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-publish) (which is actually invoked by [`lerna publish`](https://github.com/lerna/lerna/tree/main/commands/publish#lifecycle-scripts)) to build the TypeScript files. This means that as we do `lerna publish`, it will build the files beforehand for the package before it is packed and published.